app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    redirect to '/login' if !logged_in?
+    @user = current_user
+    @tweets = Tweet.all
+    erb :'tweets/index'
+  end
+
+  get '/tweets/new' do
+    redirect to '/login' if !logged_in?
+    erb :'tweets/new'
+  end
+
+  post '/tweets' do
+    if !params[:content].empty?
+      Tweet.create(user: current_user, content: params[:content])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.